resources/docs/source/includes/_structures/beatmapset_discussion_post.md

```diff
@@ -0,0 +1,16 @@
+## BeatmapsetDiscussionPost
+
+Represents a post in a [BeatmapsetDiscussion](#beatmapsetdiscussion).
+
+Field                 | Type                     | Description
+--------------------- | ------------------------ | -----------
+beatmap_discussion_id | number                   | |
```

Comment:
this should be renamed eventually...

Reply:
wonder if we should just duplicate a `beatmapset_discussion_id` field on everything

Reply:
:thinking: is it even currently used for anything